BibTeX.js

```diff
@@ -48,11 +48,11 @@ function detectImport() {
 	var chr = "";
 	var charsRead = 0;
 	
-	var re = /^\s*@[a-zA-Z]+[\(\{]/;
+	var re = /^\s*@[a-zA-Z]+[({]/;
```

Comment:
There were a bunch of regex errors reported. I fixed them all but REs are touchy. Please take a second look before committing.